test/e2e/deploy_native_test.go

```diff
@@ -33,6 +33,7 @@ func TestDeploymentOfNativeContract(t *testing.T) {
 
 		counterStart := uint64(time.Now().UnixNano())
 		contractName := fmt.Sprintf("CounterFrom%d", counterStart)
+		fmt.Println("Will attempt to deploy contract with name: ", contractName)
```

Comment:
I'd add a todo explaining why this is here and that it should be removed once resolved